repos/system_upgrade/el7toel8/actors/postgresqlcheck/libraries/library.py

```diff
@@ -0,0 +1,99 @@
+from leapp.models import InstalledRedHatSignedRPM
+from leapp import reporting
+from leapp.reporting import create_report
+from leapp.libraries.stdlib import CalledProcessError, api, run
+
+
+def get_postgresql_packages():
+    """ Get list of installed PostgreSQL packages """
+
+    for rpm_pkgs in api.consume(InstalledRedHatSignedRPM):
+        return {pkg.name for pkg in rpm_pkgs.items if pkg.name.startswith('postgresql')}
```

Comment:
Currently, only RPMs that are checked in the actor are `postgresql-server` and `postgresql-contrib`. From that point it doesn't seem necessary to use this function. Instead of that, wouldn't be better to use `has_package` function, that will provide that info? See:

```
   from leapp.libraries.common.rpms import has_package

   if has_package(InstalledRedHatSignedRPM, 'postgresql-server'):
```

We prefer this library function to be used instead, but it is not blocker. The code looks good anyway, but I haven't tested it..